### PR TITLE
Added OAK-T board.

### PR DESCRIPTION
**Pull request by zrezke**

### CODE REVIEW

**Comment by zrezke** [comment]
cc: @Luxonis-David
To check over boardConf and hardwareConf. Took them from OAK-D-SR-POE

File: batch/eeprom/MD2084_R3M2E2_oak_t.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "batchName": "",
+    "batchTime": 0,
+    "boardConf": "nIR-C80M00-00",
```
Comment (Luxonis-David) [inline batch/eeprom/MD2084_R3M2E2_oak_t.json:4]:
```suggestion
    "boardConf": "nIR-C55M11-01",
```

File: batch/eeprom/MD2084_R3M2E2_oak_t.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "boardName": "MD2084",
```
Comment (Luxonis-David) [inline batch/eeprom/MD2084_R3M2E2_oak_t.json:5]:
```suggestion
    "boardName": "MD2084-ASM",
```

File: batch/eeprom/MD2084_R3M2E2_oak_t.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    "boardRev": "R3M2E2",
```
Comment (Luxonis-David) [inline batch/eeprom/MD2084_R3M2E2_oak_t.json:6]:
```suggestion
    "boardRev": "R01-CR",
```

File: batch/eeprom/MD2084_R3M2E2_oak_t.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    "productName": "OAK-T",
+    "boardCustom": "",
+    "hardwareConf": "F1-FV00-BC000",
```
Comment (Luxonis-David) [inline batch/eeprom/MD2084_R3M2E2_oak_t.json:9]:
```suggestion
    "hardwareConf": "F1-FV04-BC000",
```

**Review by Luxonis-David — CHANGES_REQUESTED** [review]

File: batch/oak_t.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+    "title": "OAK-T",
+    "description": "OAK-T",
+    "image": "images/oak_d.jpg",
+    "options": { "bootloader": "poe", "imu": true, "usb3": false, "eth": false },
+    "variants": [
+        {
+            "title": "OAK-T (MD2084 R3M2E2)",
```
Comment (themarpe) [inline batch/oak_t.json:8]: Lets mark this here as well and in EEPROM filename, etc...
CC: @Luxonis-David @ahrastnik  that this nomenclature has to be known by KT as well

**Review by themarpe — CHANGES_REQUESTED** [review]

File: batch/oak_t.json (hunk 5 above)
Comment (themarpe) [reply batch/oak_t.json:8]: And everywhere else

**Comment by zrezke** [comment]
All has been addressed, would be nice to mainline soon. @themarpe @Luxonis-David

**Review by Luxonis-David — APPROVED** [review]

File: boards/OAK-T.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+    "board_config":
+    {
+        "name": "OAK-T",
+        "revision": "R3M1E2",
+        "cameras":{
```
Comment (themarpe) [inline boards/OAK-T.json:6]: CC: @CenekAlbl 

I assume we'll want to specify the thermal imager here as well @zrezke ?

**Review by themarpe — CHANGES_REQUESTED** [review]

File: boards/OAK-T.json (hunk 6 above)
Comment (ahrastnik) [reply boards/OAK-T.json:6]: @zrezke could you please add the missing thermal camera?

Comment (zrezke) [reply boards/OAK-T.json:6]: @ahrastnik added.

**Review by ahrastnik — APPROVED** [review]
LGTM, thanks @zrezke!

File: boards/OAK-T.json (hunk 6 above)
Comment (zrezke) [reply boards/OAK-T.json:6]: I think we need an approve of @themarpe too here, because of the requested changes.

**Review by themarpe — APPROVED** [review]
LGTM Thanks!

**Pull request merged**